src/components/notification/AposNotification.stories.js

```diff
@@ -44,13 +47,26 @@ export const notifications = () => ({
           },
           null
         )
+    },
+    progress: {
+      default: boolean('Progress Meter', false)
+    },
+    progressState: {
+      default: number('Progress State', 42, {
+        range: true,
+        min: 0,
+        max: 100
+      })
     }
   },
   template: `
-    <AposNotification 
+    <AposNotification
       :label="label"
       :type="type"
       :icon="icon"
+      :progress="progress"
+      :progressState="progressState"
+      :progressMax="100"
```

Comment:
Wrap these up into a single object? We've run into a few places where we needed more parameters and it stinks to hunt and peck for the places to add more props.

Reply:
Also i would just bake in 100 as the max, no need to alter